Astik87/src/routes.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { Layout } from "./components/Layout"
+import { Messenger } from './components/Messenger'
+import { Profile } from './components/Profile';
+
+export const routes = [
+    {
+        path: '/',
+        exact: true,
+        render: () => {return <Layout content={<Messenger />} />; } 
```

Comment:
И файлу расширение jsx тогда
```suggestion
        render: () => <Layout content={<Messenger />} />; 
```